packages/babel-preset/src/web.ts

```diff
@@ -21,15 +34,15 @@ const getDefaultPresetEnvOption = (options: WebPresetOptions) => {
 };
 
 export const getBabelConfigForWeb = (options: WebPresetOptions) => {
-  const mergedOptions = deepmerge(
```

Comment:
The action that remove deepmerge may change the behavior？

Reply:
The behavior is the same as before, `getDefaultPresetEnvOption` only returns these values:

![image](https://github.com/web-infra-dev/rsbuild/assets/7237365/1bb989df-65cf-497c-8749-45f08dd9493b)

`coreJs` can use object.assign to merge, as the properties are all required:

![image](https://github.com/web-infra-dev/rsbuild/assets/7237365/bf356dba-17c0-40df-baab-9b27a36cc9bb)

Reply:
nice